Dockerfile.dapper

```diff
@@ -14,46 +14,60 @@ RUN apt-get update && \
         psmisc \
         python-tox
 
-# Install Go 1.6
-RUN curl -o go.tar.gz https://storage.googleapis.com/golang/go1.6.2.linux-amd64.tar.gz
-RUN echo 'e40c36ae71756198478624ed1bb4ce17597b3c19d243f3f0899bb5740d56212a  go.tar.gz' | sha256sum -c && \
-    tar xzf go.tar.gz -C /usr/local && \
-    rm go.tar.gz
-RUN mkdir -p /go
-ENV PATH $PATH:/usr/local/go/bin
-ENV GOPATH=/go
+# Install Go
+ENV GOVERSION 1.6.2
+ENV GOSHA256 e40c36ae71756198478624ed1bb4ce17597b3c19d243f3f0899bb5740d56212a
+
+RUN curl -o go${GOVERSION}.linux-amd64.tar.gz https://storage.googleapis.com/golang/go${GOVERSION}.linux-amd64.tar.gz \
+    && echo "${GOSHA256} go${GOVERSION}.linux-amd64.tar.gz" | sha256sum --check \
+    && tar -xzf go${GOVERSION}.linux-amd64.tar.gz -C /usr/local \
+    && rm go${GOVERSION}.linux-amd64.tar.gz \
+    && mkdir -p /go
+
+ENV GOROOT /usr/local/go
+ENV GOPATH /go
+ENV PATH $PATH:${GOROOT}/bin:${GOPATH}/bin
 
 # Go tools
-RUN go get github.com/rancher/trash
-RUN go get github.com/golang/lint/golint
+RUN go get github.com/rancher/trash \
+    && go get github.com/golang/lint/golint
```

Comment:
Please fragment dependency installations to as many `RUN`s as possible like it was being done since that is more `build cache` friendly, helps identify breaking points in build process, helping the docker daemon pick up from where you left off.